pkgs/top-level/perl-packages.nix

```diff
@@ -2327,6 +2327,37 @@ let
      };
   };
 
+  ClusterSSH = buildPerlPackage rec {
+    name = "App-ClusterSSH-4.13.2";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/D/DU/DUNCS/App-ClusterSSH-v4.13.2.tar.gz;
+      sha256 = "0rmk2p3f2wz1h092anidjclh212rv3gxyk0c641qk3frlrjnw6mp";
+    };
+    propagatedBuildInputs = [ ExceptionClass FilePath GetoptLong LocaleMaketext Tk TryTiny X11Protocol X11ProtocolOther pkgs.coreutils pkgs.openssh pkgs.xterm ];
+    buildInputs = [ FileSlurp FileTemp FileWhich CPANChanges TestPerlTidy TestPod TestPodCoverage Readonly TestDifferences TestDistManifest TestTrap ];
```

Comment:
Please break both lists into multiple lines for readability.

Reply:
Done

Reply:
@Mic92: these lists are managed by ```cpan2nix``` so it will sort them and put into single lines on the next run (and would undo any manual changes which are not in the exception list).
I see the problem with readability and probably ```cpan2nix``` should address it, but perhaps it would be better to tag somehow the parts of file as bot-generated.

Reply:
ok. I was not aware of that.